index.html

```diff
@@ -194,7 +200,7 @@
 
                         <h4>Who is eligible?</h4>
 
-                        <p>This training program is open to you if you are between the ages of 15-29 and self-identify as having an invisible disability (for example, a learning disability or any other condition that may not be apparent to others) or an episodic disability (for example, MS, lupus, or any other condition where you have good days and bad days). We do not require a doctor’s note or other documents. No prior experience is necessary. However, basic literacy and familiarity with windows operating systems and computers would be an asset.</p>
+                        <p>This training program is open to you if you are between the ages of 15-29 and self-identify as having an invisible disability (for example, a learning disability or any other condition that may not be apparent to others) or an episodic disability (for example, MS, lupus, or any other condition where you have good days and bad days). We do not require a doctors note or other documents. No prior experience is necessary. However, basic literacy and familiarity with windows operating systems and computers would be an asset.</p>
```

Comment:
This apostrophe should remain, I think, if the interpretation is "a note from a doctor"